backend/telemetry_core/src/aggregator/inner_loop.rs

```diff
@@ -100,6 +101,30 @@ pub enum FromFeedWebsocket {
     Disconnected,
 }
 
+/// A set of metrics returned when we ask for metrics
+#[derive(Clone, Debug, Default)]
+pub struct Metrics {
+    /// When in unix MS from epoch were these metrics obtained
+    pub timestamp_unix_ms: u64,
+    /// How many chains are feeds currently subscribed to.
+    pub chains_subscribed_to: usize,
+    /// How many feeds are currently subscribed to something.
+    pub subscribed_feeds: usize,
+    /// How many feeds have asked for finality information, too.
```

Comment:
What "too" was hinting at is that it's the number of feeds that are subscribed to a chain and also have asked for finality (because you can't ask for finality without being subscribed)

Reply:
I see. How about this: "Number of subscribed feeds that also asked for finality information."?

Reply:
I like it!